pkg/Microsoft.NETFramework.Compatibility/Configurations.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp;
```

Comment:
Really?  Typically I mark the packages for "package" configuration which only builds during AllConfiguration leg.  I guess since it's only a meta-package it doesn't matter so much, but a lot of the things its referencing only build in the AllConfiguration leg.